genomic-data-product.md

```diff
@@ -109,3 +112,41 @@ Download Sorghum bicolor v3.1 from [Phytozome](https://phytozome.jgi.doe.gov/pz/
 `java –Xmx8g –jar GenomeAnalysisTK.jar –T HaplotypeCaller –R Sbicolor_313_v3.0.fa –I SAMPLEA.dedup.realigned.bam --emitRefConfidence GVCF --pcr_indel_model NONE -o SAMPLEA.output.raw.snps.indels.g.vcf`
 
 
+
+### **Above this point is the workflow for the creation of the gVCF files for this project.  The following additional steps were used to create the Hapmap file
+
+
+
+### **Combining gVCFs with GATK CombineGVCFs**
+## **NOTE: This project has 363 gvcfs: multiple instances of CombineGVCFs, with unique subsets of gvcf files, were run in parallel to speed up this step below are examples** 
+
+`java –Xmx8g –jar GenomeAnalysisTK.jar –T CombineGVCFs –R Sbicolor_313_v3.0.fa -V SAMPLEA.output.raw.snps.indels.g.vcf --variant SAMPLEB.output.raw.snps.indels.g.vcf -V SAMPLEC.output.raw.snps.indels.g.vcf -o SamplesABC_combined_gvcfs.vcf`
+
+`java –Xmx8g –jar GenomeAnalysisTK.jar –T CombineGVCFs –R Sbicolor_313_v3.0.fa --variant SAMPLED.output.raw.snps.indels.g.vcf -V SAMPLEE.output.raw.snps.indels.g.vcf -V SAMPLEF.output.raw.snps.indels.g.vcf -o SamplesDEF_combined_gvcfs.vcf`
+
+
+
+## **Joint genotyping on gVCF files with GATK GenotypeGVCFs**
+
+`java –Xmx8g –jar GenomeAnalysisTK.jar –T GenotypeGVCFs –R Sbicolor_313_v3.0.fa -V SamplesABC_combined_gvcfs.vcf -V SamplesDEF_combined_gvcfs.vcf -o all_combined_Genotyped_lines.vcf`
+
+
+
+## **Applying hard SNP filters with GATK VariantFiltration**
+`java –Xmx8g –jar GenomeAnalysisTK.jar –T VariantFiltration –R Sbicolor_313_v3.0.fa -V all_combined_Genotyped_lines.vcf -o all_combined_Genotyped_lines_filtered.vcf --filterExpression "QD < 2.0" --filterName "QD" --filterExpression "FS > 60.0" --filterName "FS" --filterExpression "MQ < 40.0" --filterName "MQ" --filterExpression "MQRankSum < -12.5" --filterName "MQRankSum" --filterExpression "ReadPosRankSum < -8.0" --filterName "ReadPosRankSum"`
+
+
+
+## **Filter and recode VCF with VCFtools**
+`vcftools --vcf all_combined_Genotyped_lines_filtered.vcf --min-alleles 2 --max-alleles 2 --out all_combined_Genotyped_lines_vcftools.filtered.recode.vcf --max-missing 0.2 --recode`
+
+
+
+## **Adapt VCF for use with Tassel5**
+`tassel-5-standalone/run_pipeline.pl -Xms75G -Xmx265G -SortGenotypeFilePlugin -inputFile all_combined_Genotyped_lines_vcftools.filtered.recode.vcf -outFile all_combined_Genotyped_lines_vcftools.filtered.recode.sorted.vcf -fileType VCF`
+
+
+
+## **Convert VCF to Hapmap with Tassel5**
+`tassel-5-standalone/run_pipeline.pl -Xms75G -Xmx290G -fork1 -vcf all_combined_Genotyped_lines_vcftools.filtered.recode.sorted.vcf -export -exportType Hapmap -runfork1
```

Comment:
There's a missing backtick on the code line

Reply:
fixed in PR #167